solutions/usaco-892.mdx

```diff
@@ -74,4 +74,35 @@ public class SleepyCowSorting {
 	
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int arr[105];
```

Comment:
change name to something else that describes what it stores

Reply:
fixed :D